python/psautohint/fdTools.py

```diff
@@ -114,15 +117,15 @@ def getFontInfo(self):
         keys = dir(self)
```

Comment:
Iterate using `for key, val in vars(self).items():` instead, and adjust the rest of the method accordingly.
I've seen `dir()` used in several other places in the afdko, and it's clearly not a good way to get to the class' properties.
This will also allow us to not have to `import types`.